guides/terminology.md

```diff
@@ -0,0 +1,89 @@
+## Terminology
+
+### Mutation Backflow
+
+Mutation backflow occurs when a child context mutates an object that is owned
+by a parent context, causing changes in the parent context.  This is analogous
+to "spooky action at a distance".
+ 
+An example negative side-effect of mutation backflow occurs when a parent
+component has already rendered when a child component mutates an object owned
+or utilized by the parent component.  This forces the parent component to need
+to re-render, and is one of several causes of what we refer to as a backtracking
+rerender.
```

Comment:
I do not think we need to coin a new term for this, nor do I think this particular term adds clarity to the phenomenon it is intended to describe. Let's see if we can stick to a pattern of

- Don't do this
- Here's why it's bad
- Do this instead

Without coming up with a special new name for each antipattern.